src/redismodule.h

```diff
@@ -623,6 +624,9 @@ REDISMODULE_API const char* (*RedisModule_CallReplyVerbatimString)(RedisModuleCa
 REDISMODULE_API RedisModuleCallReply * (*RedisModule_CallReplySetElement)(RedisModuleCallReply *reply, size_t idx) REDISMODULE_ATTR;
 REDISMODULE_API RedisModuleCallReply * (*RedisModule_CallReplyMapKey)(RedisModuleCallReply *reply, size_t idx) REDISMODULE_ATTR;
 REDISMODULE_API RedisModuleCallReply * (*RedisModule_CallReplyMapVal)(RedisModuleCallReply *reply, size_t idx) REDISMODULE_ATTR;
+REDISMODULE_API RedisModuleCallReply * (*RedisModule_CallReplyAttributeKey)(RedisModuleCallReply *reply, size_t idx) REDISMODULE_ATTR;
+REDISMODULE_API RedisModuleCallReply * (*RedisModule_CallReplyAttributeVal)(RedisModuleCallReply *reply, size_t idx) REDISMODULE_ATTR;
```

Comment:
considering attributes are inherently a map, maybe it's sufficient to document that, and let the caller use the map API?
same goes for the CallReply interface.

Reply:
What if one day we will want to change implementation for attributes?

Reply:
the spec documents that they're a map so that's not gonna change.
i suppose we'll still be able to add some flag in the reply object to denote that it's an attribute map, and we'll still be able to write different logic to handle them inside the same API.

this is just an idea, i suppose unifying them can even confuse people (until the notice the documentation that explains it).
if you don't feel it's good, we can drop it.

Reply:
I actually think that split them is better, but I will not insist, let me know if you want me to split them...